bot/main.py

```diff
@@ -0,0 +1,24 @@
+import discord
+
+intents = discord.Intents.default()
+intents.message_content = True
+
+client = discord.Client(intents=intents)
+
+@client.event
+async def on_ready():
+    print(f'We have logged in as {client.user}')
+
+@client.event
+async def on_message(message):
+    if message.author == client.user:
+        return
+
+    if message.content.startswith('$hello'):
+        await message.channel.send('Hello!')
+
+import os
```

Comment:
importing in the middle of the file is not really recommended generaly speaking import are at the top unless you know what are doing and using import inside specific functions

Reply:
Same as [this comment](https://github.com/DiscordFi/MasarykReborn/pull/11#discussion_r1539553781).